spinalcordtoolbox/deepseg/inference.py

```diff
@@ -233,13 +233,19 @@ def segment_nnunet(path_img, tmpdir, predictor, device: torch.device):
     orig_orientation = get_orientation(Image(path_img_tmp))
 
     # Get the orientation used by the model
-    # TODO: Make reorientation a model configuration parameter
-    if "SCI" in predictor.plans_manager.dataset_name:
-        model_orientation = "RPI"
-    elif "RegionBasedLesionSeg" in predictor.plans_manager.dataset_name:
-        model_orientation = "AIL"
+    # Check if predictor.dataset_json['orientation'] key exists, if so, read the orientation from there
+    # NOTE: the 'orientation' key-value pair needs to be manually added to the dataset.json file
+    # TODO: document this on our intranet (https://intranet.neuro.polymtl.ca/data/deeplearning-models.html#packaging-models)
+    if 'image_orientation' in predictor.dataset_json:
+        model_orientation = predictor.dataset_json['image_orientation']
+        print(f"Model orientation (based on dataset.json): {model_orientation}")
```

Comment:
to exactly match [#4683](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/as/4683_add_canal_segmentation_nnunet/spinalcordtoolbox/deepseg/inference.py#L235-L246) and avoid conflicts

```suggestion
        print(f"Orientation (based on dataset.json): {model_orientation}")
```